IRSA-README.md

```diff
@@ -0,0 +1,88 @@
+may need to assume identities in the other Cloud Providers.  
+
+Using Web Identities based on an OpenID Connect Provider supports such use-cases. In this article we will start with a 
+basic setup where using IAM Role for Service Accounts (IRSA) we can configure Kubernetes Service Accounts attached to 
+an EKS workload to assume IAM Role(s). This is similar to attaching an IAM Role to an EC2 instance at startup. 
+Note that this mechanism allows EKS workloads to assume roles in any AWS Account and not just the AWS account hosting 
+the EKS cluster.
+
+In the subsequent series of articles we will expand on this to demonstrate how a data plane hosted on both on-premise 
+or any cloud provider can assume an IAM role in an AWS account.
+
+While this document is meant to guide you on configuring AWS IAM Role propagation to Domino Workloads based on 
+well defined policies consistent with your Enterprise Security guidelines, please speak with your Domino Professional 
+Services contact for additional information, assistance and support.
+
+### When should you consider using IRSA with Domino
+
+  - Does your Enterprise Policy forbid the use of long lived credentials?
+    
+  - Would you like to transparently and securely propagate IAM Role to all Domino Workloads including a 
+[Domino Scheduled Job](https://docs.dominodatalab.com/en/latest/user_guide/5dce1f/schedule-jobs/) ?
+    
+  - Would you like to configure a Domino Workload with a set of IAM Roles based on a set of complex custom criteria?
+
+
+### Why does using IRSA with Domino help achieve the above goals?
+
+  - When using the IRSA support in Domino, any Domino hosted workloads can assume IAM roles in any AWS account configured to
+    trust the Domino Platform
+    
+  - IRSA support in Domino allows configuration of multiple roles for a Domino workload where the boto3 library can 
+    switch across the multiple profiles configured in the ~/.aws/config file
+    
+  - The mapping of a workload to a set of roles can be configured based on custom configurable rules. This allows for 
+    complex mappings to be supported
+
+### How does IRSA work with Domino?
+ 1 . For a Domino Workload (implemented as a K8s Pod) to be able to assume an IAM Role, it must use the 
+ `STS:AssumeRoleWithWebIdentity` API which returns temporary security credentials. But first the caller (in this case 
+ a pod) needs to be authenticated with a Web Identity Provider that is compatible with the OpenID connect (OIDC) protocol.
+
+ 2. Starting [1.12](https://kubernetes.io/docs/concepts/storage/projected-volumes/) Kubernetes allows an OIDC provider
+    to be associated with it. An EKS Cluster provides a unique OIDC provider bound to the `KubeAPIServer` server. A JWT 
+    created by this OIDC provider can be mounted as a projected volume in the Pod. This JWT is associated with the K8s 
+    Service Account associated with the workload. It is distinct from the JWT issued by the `KubeAPIServer`. This JWT 
```

Comment:
`It is distinct from the JWT issued by the `KubeAPIServer`.` I'd leave this bit out, it can confuse people since technically both tokens are issued by the kube server, just the targets are different

Reply:
Good point